ee_tests/src/specs/workshop-import-to-space.spec.ts

```diff
@@ -4,9 +4,9 @@ import {
   MainDashboardPage,
   SpaceDashboardPage,
   CleanupUserEnvPage,
-  AddToSpaceDialog,
   PageOpenMode
 } from './page_objects';
+import { AddToSpaceDialog } from './ui/space_dashboard/';
```

Comment:
why / at the end? how about export {AddToSpaceDialog} in space_dashboard.page.ts?